tests/performance/local/CMakeLists.txt

```
@@ -25,6 +25,11 @@ if(NOT HPX_WITH_SANITIZERS)
   set(start_stop_FLAGS DEPENDENCIES hpx_timing)
 endif()
 
+if(HPX_WITH_LIBCDS)
+  set(benchmarks ${benchmarks} libcds_hp_overhead)
+  set(libcds_hp_overhead_FLAGS DEPENDENCIES iostreams_component hpx_timing)
```

```suggestion
  set(libcds_hp_overhead_FLAGS DEPENDENCIES iostreams_component)
```

Linking to individual modules like that should only be done if `NOLIBS` is used.